src/app/genes/gene-overview/gene-druggability/gene-druggability.component.html

```diff
@@ -20,7 +20,7 @@ <h2>
                     </div>
                     <div class="col-md-3 col-sm-4 col-4 text-center header-score">
                         <div [ngStyle]="getBucketIconStyle(false)" [style.background-color]="getBucketBGColor(geneInfo.druggability[0].sm_druggability_bucket, 'sm')" [style.color]="getBucketTextColor(geneInfo.druggability[0].sm_druggability_bucket, 'sm')">
-                            <div class="icon-description">{{geneInfo.druggability[0].sm_druggability_bucket}}</div>
+                            <div class="icon-description">{{geneInfo.druggability[0].sm_druggability_bucket !== 100 ? geneInfo.druggability[0].sm_druggability_bucket : '&ndash;'}}</div>
```

Comment:
Recommend keeping this logic in the typescript file rather than the html.  Perhaps a helper function, like "getFriendlyBucketValue (and pass sm_druggability_bucket, abability_bucket, and safety_bucket to it).
Also in the typescript, you could define this special constant value that you're using, DEFAULT_BUCKET_VALUE=100

Reply:
That makes sense, done and done